### PR TITLE
Update README.md redirecting to elements

### PR DESCRIPTION
**Pull request by alessandro-saglimbeni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,3 @@
...
------------------
-Elements is an open source collaborative project where we work on a collection
-of experiments to more rapidly bring technical innovation to the Bitcoin ecosystem.
-
-https://github.com/ElementsProject/elementsproject.org
-
-Learn more on [the Elements Project website](https://www.elementsproject.org).
+[Moved to ElementsProject](https://github.com/ElementsProject/elements)
+=======================================================================
+Liquid is actively maintained under the [elements repo](https://github.com/ElementsProject/elements) where you can also find the [binaries for liquid](https://github.com/ElementsProject/elements/releases).
```
Comment (instagibbs) [inline README.md]: Perhaps make it clearer that the binaries going forward will be *elements* binaries, which can run on the liquid network using `-chain=liquidv1`.

Comment (alessandro-saglimbeni) [reply README.md]: Updated README, is it clearer and correct now?

Comment (allenpiscitello) [reply README.md]: lgtm

**Comment by instagibbs** [comment]
utACK https://github.com/Blockstream/liquid/pull/34/commits/ee654b12e680ad1d91a2e9646d386685147a558b

**Pull request merged**